### DISCUSSION TITLE
来一场华丽的舞会吧♪♪♪

### DISCUSSION THREAD

**Discussion by EhuanyingK** [share121/inter-knot#1373] [其他]
来一场华丽的舞会吧♪♪♪打不着哦♪♪♪这里这里♪♪♪别心急嘛♪♪♪射中你的心♪♪♪逃不掉哦♪♪♪我的心意，要好好收下哦♪♪♪浪漫的共演哦♪♪♪嗨～这之后的时间，也要继续加油哦♪♪♪
飞向远天吧♪♪♪以我为始，以我为终♪♪♪星星划过的时候，要记得许愿哦♪♪♪璀璨光芒，照耀大地♪♪♪你很热情嘛♪♪♪要踩好节奏哦♪♪♪致以，无瑕的时代♪♪♪哇，好高好高♪♪♪小心脚下♪♪♪魔法的时间，结束啦♪♪♪

![Image_1722950494202](https://github.com/user-attachments/assets/7edb268f-c81b-4f9d-b17e-877bced7aa27)

**Comment by gaojiaobei** [comment]
爱门！

**Comment by llxcx4** [comment]
**爱门!!!!!**

**Comment by qepe** [comment]
### 爱门!

**Comment by Asaki-333** [comment]
爱莉占领绳网

**Comment by March3th** [comment]
<div align="center"><h1>爱门</h1></div>
[TRUE]

T——To
R——Romantic浪漫
U——Unfailing始终如一
E——Elysia爱莉希雅
TRUE——To Romantic Unfailing Elysia
致浪漫的始终如一的爱莉希雅
（转自b站高赞评论)

**Comment by Loxia1121** [comment]
✝爱门✝

**Comment by Myitian** [comment]
![8b7e9b8296aa166fcd24e04165ff1826_5967180535634639907](https://github.com/user-attachments/assets/4bdf35bd-f2f4-498c-b4f6-2ce36fd8e11a)

**Comment by 12344856** [comment]
爱门！
